src/commands/install.yml

```diff
@@ -17,7 +17,9 @@ steps:
 
         if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
 
-        $SUDO apt-get update && $SUDO apt-get -qqy install apt-transport-https
+        # https://github.com/CircleCI-Public/azure-cli-orb/issues/15
+        # https://manpages.debian.org/unstable/apt/apt-get.8.en.html
+        $SUDO apt-get --allow-releaseinfo-change update && $SUDO apt-get -qqy install apt-transport-https
```

Comment:
While this works, I believe it's better to be specific about what we allow. This way, we don't hide unexpected changes. That said, what do you think of replacing `--allow-releaseinfo-change-suite` with `--allow-releaseinfo-change`?

You can read more about it here: https://stackoverflow.com/a/68990841.